tc/core/polyhedral/codegen.h

```diff
@@ -27,14 +28,13 @@ class Codegen {
 
  public:
   // Create a list of isl ids to be used as loop iterators when building the
-  // AST.
+  // AST from the schedule tree with the given root.
   //
   // Note that this function can be scrapped as ISL can generate some default
```

Comment:
Looking at this comment you did not touch (sorry for that), is there indeed such a function to generate a list of ids with the same prefix and different numeric suffixes?  I would expect it in PPCG.  If so, let's use that function.  Otherwise, let's kill the incorrect comment since we are touching the function.